tools/upgrade_versions.py

```diff
@@ -232,9 +238,11 @@ def run(CONF):
     for pkg in pm.packages:
         pkg.lock()
         pkg_version = Version(pkg)
-        pkg_version.update(committer_name, committer_email)
+        pkg_version.update(updater_name, updater_email)
         pkg.unlock()
 
-    release_date = datetime.today().date().isoformat()
-    push_new_versions(versions_repo, release_date, push_repo_url,
-                      push_repo_branch, committer_name, committer_email)
+    if commit_updates:
+        release_date = datetime.today().date().isoformat()
+        commit_and_push_new_versions(versions_repo, release_date, push_repo_url,
+                                     push_repo_branch, updater_name, updater_email,
+                                     push_updates)
```

Comment:
I think it would be better to divide this function in two to reduce the number of parameters. I suggest:
`commit_new_versions(versions_repo, release_date, updater_name, updater_email)`
`push_head_commit(versions_repo, push_repo_url, push_repo_branch)`